prql-compiler/src/error.rs

```diff
@@ -115,14 +115,14 @@ impl StdError for Errors {}
 // Needed for StdError
 impl Display for Error {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        Debug::fmt(&self, f)
+        Display::fmt(&self, f)
```

Comment:
This will recurse on itself. But it was probably never called, because it is converted to ErrorMessage, before it is displayed.